thirdparty/fetch-thirdparty-deps.sh

```diff
@@ -52,7 +52,7 @@ if [ ! -f MaxMind.GeoIP2.dll ]; then
 	get MaxMind.Db 1.0.0.0
 	get Newtonsoft.Json 6.0.5
 	get RestSharp 105.0.1
```

Comment:
You need to update those as well. They are not automatically resolved and manually specified as noget has no dependency resolution of it's own.